cmd/run/flags.go

```diff
@@ -182,4 +182,13 @@ func bindRunFlags(command *cobra.Command) {
 	flags.Uint32("listObjects-max-results", defaultConfig.ListObjectsMaxResults, "the maximum results to return in ListObjects responses")
 	util.MustBindPFlag("listObjectsMaxResults", flags.Lookup("listObjects-max-results"))
 	util.MustBindEnv("listObjectsMaxResults", "OPENFGA_LIST_OBJECTS_MAX_RESULTS", "OPENFGA_LISTOBJECTSMAXRESULTS")
+
+	flags.Bool("allow-writing-10-models", defaultConfig.AllowWriting10Models, "allow writing of schema 1.0 models")
```

Comment:
This flag is formatted confusingly. `10-models` comes across as "10" (the literal ten) instead of "1.0". 

I may recommend introducing the period for accuracy or maybe it would suffice to just have `allow-writing-deprecated-models` and `allow-evaluating-deprecated-models` which disallows writing and evaluation deprecated model schema versions (which right now only refers to 1.0 models).